Taxes/Integration/02_ManageTaxClasses.md

```diff
@@ -0,0 +1,106 @@
+[!!Accounting](RetailSuiteAccounting)  
+[!!Fakturierung](RetailSuiteFaktBase)  
+[!!User Interface Tax classes](../UserInterface/01c_TaxClasses.md)  
+[!!Manage the tax rates](./01_ManageTaxRates.md)  
+
+
+
+# Manage the tax classes
+
+Each product or group of products is assigned a tax class. The tax class is determined by the competent tax authorities of the territory where the product or group of products is to be taxed. Therefore it usually differs from territory to territory.
+
+There are two main tax classes in most territories: the standard and the reduced tax rate class. However, there can also be special taxes, such as super-reduced or zero taxes. Tax classes can be created, edited and deleted if necessary.
+
+
+## Create a tax class
+
+Create a tax class to be assigned to a product or group of products.
+
+#### Prerequisites  
+
+No prerequisites to fulfill.
+
+#### Procedures
+
+*Taxes > Settings > Tab TAX CLASSES*
+
+![Tax classes](../../Assets/Screenshots/Taxes/Settings/TaxClasses/TaxClasses.png "[Tax classes]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the upper left corner.   
+  A new data entry line is displayed.  
+
+2. Click the entry field to be edited.  
+  The field becomes active and the cursor is displayed.
+
+3. Enter the desired tax class name in the entry line.  
+
+4. Click the [SAVE] button.  
+
+5. Refresh the *Tax* module for the *TAX RATES* tab to display the changes made. To do so, click the F5 key.  
+The newly created tax class is displayed in the *Tax class* drop-down list in the *TAX RATES* tab.
```

Comment:
```suggestion
5. Press **F5** to initialize the *Core1 Platform* and to apply the changes.   
The new tax class is displayed in the list of tax classes.
```
Das ist aber ganz schön umständlich, wenn ich erst in nen anderes Tab gehen muss und da einen neuen Steuersatz anlegen oder bearbeiten muss, damit ich sehen, dass die Steuerklasse angelegt wurde...